src/layouts/content-page.vue

```diff
@@ -42,14 +43,21 @@
 </template>
 
 <script setup>
-  import query from './default.query.graphql?raw';
+  import query from './content-page.query.graphql?raw';
 
   const { $i18n } = useNuxtApp();
   const { afterEach } = useRouter();
   const skipLink = ref(null);
   const mobileMenuIsOpen = ref(false);
   const focusTrapMobileMenu = computed(() => mobileMenuIsOpen.value ? true : null);
 
+  defineProps({
+    hideFooter: {
+      type: Boolean,
+      default: false,
+    },
+  });
```

Comment:
Let's use the TS native types like so: `defineProps<{}>()`

Reply:
As discussed, skipping this for now.